src/cast_core/src/CrudActor.cpp

```diff
@@ -1273,8 +1286,12 @@ struct DropOperation : public BaseOperation {
 std::unordered_map<std::string, OpCallback&> getOpConstructors() {
     // Maps the yaml 'OperationName' string to the appropriate constructor of 'BaseOperation' type.
     std::unordered_map<std::string, OpCallback&> opConstructors = {
-        {"aggregate", baseCallback<BaseOperation, OpCallback, AggregateOperation>},
         {"bulkWrite", collectionHandleCallback<BaseOperation, OpCallback, BulkWriteOperation>},
+        {"withTransaction",
```

Comment:
It appears that you didn't actual change these lines but just moved them around. May I know the reason of adjusting the order of this list?

Reply:
I wanted to group the op constructors that use `collectionHandleCallback` ("bulkWrite" and "withTransaction") closer together, while also keeping the transaction-related op constructors close together.

Reply:
I wanted to group the op constructors that use `collectionHandleCallback` ("bulkWrite" and "withTransaction") closer together, while also keeping the transaction-related op constructors close together.